app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="column content-box">
```

Comment:
@vishnu-narayanan Is it possible to relocate the footer to the bottom of the screen? Currently, it appears to be shifting whenever we navigate to a different page, causing a jarring experience.

```suggestion
  <div class="column content-box audit-log--settings">
```

Add this to the bottom as style tag

```
<style lang="scss" scoped>
.audit-log--settings {
  display: flex;
  justify-content: space-between;
  flex-direction: column;

  .remote-address {
    width: 14rem;
  }

  .wrap-break-words {
    word-break: break-all;
    white-space: normal;
  }
  .column-activity {
    width: 60%;
  }
}
</style>
```